templates/default/snapraid_runner.conf.erb

```diff
@@ -1,14 +1,19 @@
+<%
+path_separator = node['platform'] == 'windows' ? '\\' : '/'
+executable_extension = node['platform'] == 'windows' ? '.exe' : ''
+executable = node['snapraid']['install_directory'] + path_separator + 'snapraid' + executable_extension
+%>
 [snapraid]
 ; path to the snapraid executable (e.g. /bin/snapraid)
-executable = /usr/local/bin/snapraid
+executable = <%=executable%>
```

Comment:
It would be cleaner, but on the other hand it would kind of be parametrising something that isn't truly customisable. If you see something in attribs you expect to be able to change it, but you can't meaningfully change`default['snapraid']['executable']` as you'd also have to change node['snapraid']['install_directory']` appropriately to effectively reflect the same.